versions.tf

```diff
@@ -1,3 +1,9 @@
 terraform {
-  required_version = ">= 0.12.0"
-}
+  required_version = ">= 0.13.0"
+
+  required_providers {
+    tfe = {
+      source = "hashicorp/tfe"
+    }
+  }
+}
```

Comment:
Please make sure to end the file with a newline. Probably best to configure your editor to automatically insert a newline at the end of the file.